shopsdk/install.py

```diff
@@ -0,0 +1,278 @@
+	  zu.invalidexit()
+	elif ar==4: #Mode 4 Identifier Software Check
+          print("Reserved for primary storage file reading, WIP")
+	  zu.invalidexit()
+	elif ar==5: #Mode 5 Identifier Software Check
+	  f=open('id.txt', z)
+	  ar_contents=f.read()
+	  ap(ar_contents)
+	  f.close()
+	  av()
+	elif ar==6: #Mode 6 Identifier Software Check
+	  ar=open('tables.txt')
+	  aq=ar.readlines()
+	  ap((aq[al]))
+	  av()
+	elif ar==7: #Mode 7 Identifier Software Check
+	  ap(os.getcwd()) #Grabs Current Path
+	  av()
+	elif ar==8: #Mode 8 Identifier Software Check
+	  az=list(range(0, 60)) # Returns Preset Line Number Range
+	  am=open(an, y)
+	  for line in az:
+	    am.write(str(line))
+	    am.write(aw)
+	  am.close()
+	  a0 = "nwcs_path_data\n"
+	  a1 = "pwinty_use_private_key\n"
+	  a2 = "pwinty_execute_all\n"
+	  a3 = "photos_use_private_key\n"
+	  a4 = "photos_execute_all\n"
+	  a5 = "printful_use_private_key\n"
+	  a6 = "printful_execute_all\n"
+	  a7 = "digital_use_private_key\n"
+	  a8 = "digital_execute_all\n"
+	  a9 = "nwcs_use_private_key\n"
+	  b0 = "nwcs_execute_all\n"
+	  b1 = "accf_use_private_key\n"
+	  b2 = "accf_execute_all\n"
+	  b3 = "pwinty_production\n"
+	  b4 = "pwinty_send_stats\n"
+	  b5 = "pwinty_send_logs\n"
+	  b6 = "pwinty_load_options_file\n"
+	  b7 = "pwinty_use_pickle\n"
+	  b8 = "photos_production\n"
+	  b9 = "photos_send_stats\n"
+	  c0 = "photos_send_logs\n"
+	  c1 = "photos_load_options_file\n"
+	  c2 = "photos_use_pickle\n"
+	  c3 = "digital_production\n"
+	  c4 = "digital_send_stats\n"
+	  c5 = "digital_send_logs\n"
+	  c6 = "digital_load_options_file\n"
+	  c7 = "digital_use_pickle\n"
+	  c8 = "printful_production\n"
+	  c9 = "printful_send_stats\n"
+	  d0 = "printful_send_logs\n"
+	  d1 = "printful_load_options_file\n"
+	  d2 = "printful_use_pickle\n"
+	  d3 = "nwcs_production\n"
+	  d4 = "nwcs_send_stats\n"
+	  d5 = "nwcs_send_logs\n"
+	  d6 = "nwcs_load_options_file\n"
+	  d7 = "nwcs_use_pickle\n"
+	  d8 = "accf_production\n"
+	  d9 = "accf_send_stats\n"
+	  e0 = "accf_send_logs\n"
+	  e1 = "accf_load_options_file\n"
+	  e2 = "accf_use_pickle\n"
+	  e3 = "pwinty_sentry_url\n"
+	  e4 = "photos_sentry_url\n"
+	  e5 = "printful_sentry_url\n"
+	  e6 = "digital_sentry_url\n"
+	  e7 = "nwcs_sentry_url\n"
+	  e8 = "accf_sentry_url\n"
+	  e9 = "printful_cdn_url\n"
+	  f0 = "pwinty_cdn_url\n"
+	  f1 = "accf_cdn_url\n"
+	  f2 = "datadog_api_key\n"
+	  f3 = "pwinty_app_key\n"
+	  f4 = "photos_app_key\n"
+	  f5 = "printful_app_key\n"
+	  f6 = "digital_app_key\n"
+	  f7 = "pwinty_api_key\n"
+	  f8 = "printful_api_key\n"
+	  f9 = "0\n"
+	  l0 = a0 + a1 + a2 + a3 + a4 + a5 + a6 + a7 + a8 + a9
+	  l1 = b0 + b1 + b2 + b3 + b4 + b5 + b6 + b7 + b8 + b9
+	  l2 = c0 + c1 + c2 + c3 + c4 + c5 + c6 + c7 + c8 + c9
+	  l3 = d0 + d1 + d2 + d3 + d4 + d5 + d6 + d7 + d8 + d9
+	  l4 = e0 + e1 + e2 + e3 + e4 + e5 + e6 + e7 + e8 + e9
+	  l5 = f0 + f1 + f2 + f3 + f4 + f5 + f6 + f7 + f8 + f9
+	  zz = l0 + l1 + l2 + l3 + l4 + l5
```

Comment:
Why is this necessary? What is it used for?

Reply:
That was scrapped, it was a bit of a amibitous feature that would've been needed with v1, but I don't need it anymore.

Reply:
That was for scrapped feature, v1 needed it but this new version doesn't :)